kernel/byterun/dune

```diff
@@ -5,7 +5,7 @@
  (foreign_stubs
   (language c)
   (names coq_fix_code coq_float64 coq_memory coq_values coq_interp)
-  (flags :standard (:include %{project_root}/config/dune.c_flags))))
+  (flags -fPIC (:include %{project_root}/config/dune.c_flags))))
```

Comment:
Why is this `-fPIC` needed?

Reply:
@silene I added it following your comment in https://github.com/coq/coq/pull/14519#issuecomment-869822279 , maybe I misunderstood what "forced on us" mean.

Reply:
What I meant is that, as of today, C code meant to be linked with OCaml code needs to be compiled with `-fPIC`. Assuming that Dune knows how to compile C code (does it?), we should never have to pass it explicitly.

Reply:
Dune knows how to compile C code but have explicitly asked it not to interfere with our choice of flags , `(use_standard_c_and_cxx_flags false)` , so in a sense we can't have our cake and eat it.

I recall the set of flags that dune uses as `:standard` : `-fno-strict-aliasing -fwrapv -fPIC`

Reply:
Another way to express stuff like this is `(flags :standard \ -fwrapv (:include ...))` , YMMV.

Reply:
I don't understand what you are saying. In the build log you have shown, option `-fPIC` is passed twice, so there is just no reason to pass it manually. Moreover, `-fwrapv` is being passed too, so Dune's standard flags are being passed, even if you didn't use `:standard`.

Reply:
Oh yes, sorry, the value for the option is actually wrong; PR fixed. Effective flags are now:
```
/usr/bin/gcc -fPIC -Wall -Wno-unused -g -O2 -msse2 -mfpmath=sse -g -I /home/egallego/.opam/dev-coq-lsp/lib/ocaml -o coq_interp.o -c coq_interp.c
```

Reply:
ping @silene